### ISSUE TITLE
Decred forum

### ISSUE THREAD

**Issue by dcrlove2021** [xaur/decred-issues#152]
<img width="1408" alt="image" src="https://user-images.githubusercontent.com/82247290/205474396-f7aae54f-7605-4255-9ac9-b043e40f63d5.png">

# If there is a dcr forum like Bitcoin Forum, it should be very interesting，More interaction than just columnists

**Comment by PhoenixGreen** [comment]
A forum sounds like an interesting idea. Ghost doesn't come preloaded with a forum but if someone starts one I'm happy to link to it. Although, the Decred Reddit area could be a good fit for this and it already exists.

**Comment by xaur** [comment]
There is a Comments section under Decred Magazine posts but I never tried if it works.

Our forum story is pretty sad so far. We had a self-hosted forum but it was frozen to consolidate all forum-like comms on Reddit. Then we learned more about Reddit and some community members avoid posting on Reddit now (see #38). Currently most interesting conversations take place in our chats in Matrix, Discord, and Telegram.

A Reddit-like forum built on top of Politeia (#38) was anticipated for quite some time but it is taking long and at this pace I would not expect it any time soon.

If anyone launches a forum that would be nice I guess, although it will take some creativity to onboard people in it.

So what you can do currently is post [on our Reddit](https://www.reddit.com/r/decred/new/) (if it is available in China?) or in our chats (https://chat.decred.org).

**Comment by PhoenixGreen** [comment]
Yeah, the comment section in Decred Magazine is very good and lightweight. One thing I like about it is comments are article specific which means you can have some interesting conversations all on topic.

I’ve run forums in the past and generally the experience was pretty terrible.

**Comment by dcrlove2021** [comment]
Yes, I have used Cred Reddit, but it doesn't seem to be applicable to China. China needs vpn to access it, and  no Chinese, 
 Chinese  people won't use it without Chinese!

**Comment by dcrlove2021** [comment]
It would be interesting if Politia could be deployed independently without registration fee!

**Comment by dcrlove2021** [comment]
Politia ui. There is obviously a threshold for a new comer, because he needs to have dcrcoin first

**Comment by xaur** [comment]
Politeia uses the registration fee to prevent registration spam. Indeed it is a barrier for new users but this is what we have now.

In theory it may be possible to improve this, but mind that Politeia development is not too active recently so I would not expect this in the short term.
